>From e23a4a8a79b377b273a4c7e30e8f714c85308cc2 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Mon, 18 May 2015 15:35:48 +0900
Subject: [PATCH] add backports as comment out list

once backports was disabled (see Bug#764982) for sane default install but
adding backports apt source list as sample comment out one is better for
users who want to add backports. It's handy.
---
 debian/apt-setup-udeb.templates | 2 +-
 generators/93backports          | 8 ++++----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/debian/apt-setup-udeb.templates b/debian/apt-setup-udeb.templates
index b106bf2..3f54087 100644
--- a/debian/apt-setup-udeb.templates
+++ b/debian/apt-setup-udeb.templates
@@ -84,7 +84,7 @@ Choices-C: security, updates, backports
 # including host name. In short, KEEP THIS SHORT and, yes, that's tricky
 # :sl1:
 __Choices: security updates (from ${SEC_HOST}), release updates, backported software
-Default: security, updates
+Default: security, updates, backports
 # :sl1:
 _Description: Services to use:
  Debian has two services that provide updates to releases: security and
diff --git a/generators/93backports b/generators/93backports
index 1021d5f..5d9c218 100755
--- a/generators/93backports
+++ b/generators/93backports
@@ -41,13 +41,13 @@ echo "# ${codename}-backports, previously on backports.debian.org" >> $file
 
 # Don't test mirror if no network selected in netcfg
 if  [ -n "$protocol" ] && [ -n "$host" ]; then
-	echo "deb $protocol://${host}${directory} ${codename}-backports $dists" >> $file
+	echo "#deb $protocol://${host}${directory} ${codename}-backports $dists" >> $file
 else
 	echo "# A network mirror was not selected during install.  The following entries" >> $file
 	echo "# are provided as examples, but you should amend them as appropriate" >> $file
 	echo "# for your mirror of choice." >> $file
 	echo "#" >> $file
-	echo "# deb http://ftp.debian.org/debian/ ${codename}-backports $dists" >> $file
+	echo "#deb http://ftp.debian.org/debian/ ${codename}-backports $dists" >> $file
 fi
 if db_get netcfg/dhcp_options && \
    [ "$RET" = "Do not configure the network at this time" ]; then
@@ -70,9 +70,9 @@ else
 fi
 
 if [ -n "$protocol" ] && [ -n "$host" ]; then
-	echo "deb-src $protocol://${host}${directory} ${codename}-backports $dists" >> $file
+	echo "#deb-src $protocol://${host}${directory} ${codename}-backports $dists" >> $file
 else
-	echo "# deb-src http://ftp.debian.org/debian/ ${codename}-backports $dists" >> $file
+	echo "#deb-src http://ftp.debian.org/debian/ ${codename}-backports $dists" >> $file
 fi
 
 exit $CODE
-- 
2.1.4

